mne/viz/topomap.py

```diff
@@ -1613,3 +1606,17 @@ def _find_peaks(evoked, npeaks):
     if len(times) == 0:
         times = [evoked.times[gfp.argmax()]]
     return times
+
+
+def _prepare_topomap(pos, ax):
+    """Helper for preparing the topomap."""
+    pos_x = pos[:, 0]
+    pos_y = pos[:, 1]
+
+    ax.set_xticks([])
+    ax.set_yticks([])
+    ax.set_frame_on(False)
+    if any([not pos_y.any(), not pos_x.any()]):
+        raise RuntimeError('No position information found, cannot compute '
```

Comment:
This error is a bit vague for the user. I think it would be better if you explicitly say that the problem occurred when plotting the legend for the spatial_colors